### PR TITLE
feat: Support `ordered-imports` option

### LINKED ISSUE #647: Consider using an eslint import plugin instead of import/order
### 🚀 Feature Request

The script swaps in `import/order` for tslint's `ordered-imports`. I spent quite a while trying to configure it to sort things like tslint's rule did, before discovering [eslint-plugin-simple-import-sort](https://github.com/lydell/eslint-plugin-simple-import-sort), which seems much closer to the old rule (and also much easier to configure!).

#### Existing Behavior

`ordered-imports` becomes `import/order`.

#### Change Proposal

`ordered-imports` becomes `eslint-plugin-simple-import-sort`

I haven't taken the time to check how well this would work for `tslint.json` settings beyond the ones I use, but for me simple-import-sort was fantastic!

### PR DESCRIPTION
**Pull request by hyperupcall**
## PR Checklist

-   [x] Addresses an existing issue: Fixes #647
-   [x] That issue was marked as [`status: accepting prs`](https://github.com/typescript-eslint/tslint-to-eslint-config/labels/status%3A%20accepting%20prs)

## Overview

Simply adds support for the `ordered-imports` option. This is a draft PR because the correct functionality depends on a [PR within `eslint-plugin-import`](https://github.com/import-js/eslint-plugin-import/pull/2395)

### CODE REVIEW

**Comment by hyperupcall** [comment]
Just an update on this PR - My [bugfix for `eslint-plugin-import`](https://github.com/import-js/eslint-plugin-import/commit/998655b1438c9718403ae876163b739c57f658ca) just got merged, so once there is a new release, i'll make the final modifications required to undraft this PR :tada:

**Comment by hyperupcall** [comment]
Wowzies! I think this thing is ready for review again - glad this can finally be closed soon :sweat_smile: 

According to [GitHub](https://github.com/import-js/eslint-plugin-import/pull/2395/commits/998655b1438c9718403ae876163b739c57f658ca), my `eslint-plugin-import` change got released in `v2.27.0`, which was released on [January 11](https://github.com/import-js/eslint-plugin-import/commit/01950df8ec68824f52aca43b751621059fea51cd), so everyone should have an update-to-date version when they install the package after the config conversion.

File: src/converters/lintConfigs/rules/ruleConverters/ordered-imports.ts [hunk 1]
```diff
@@ -1,21 +1,124 @@
...
+            break;
+    }
+
+    if (argument["grouped-imports"] === true) {
+        importOrderRule["newlines-between"] = "always";
+    }
+
+    if ("groups" in argument) {
+        notices.push(
+            "Option 'groups' is too bespoke to be converted to ESLint plugin 'eslint-plugin-import'",
```
Comment (JoshuaKGoldberg) [inline src/converters/lintConfigs/rules/ruleConverters/ordered-imports.ts:100]: 😂 I love this

**Review by JoshuaKGoldberg — APPROVED** [review]
What a blast from the past! This is looking great, thanks @hyperupcall!

Congrats on getting this merged after all these months 😄

**Comment by JoshuaKGoldberg** [comment]
Should be published & available in [2.14.0](https://www.npmjs.com/package/tslint-to-eslint-config/v/2.14.0). Thanks @hyperupcall! 🙌

**Comment by hyperupcall** [comment]
Yay! Happy to help :smiley:

**Pull request merged**